eng/config/PublishData.json

```diff
@@ -1,4 +1,25 @@
 {
+  "feeds": {
+    "vs-impl": "https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl/nuget/v3/index.json"
+  },
+  "packages": {
+    "default": {
+      "rzls": "vs-impl",
+      "rzls.alpine-arm64": "vs-impl",
+      "rzls.alpine-x64": "vs-impl",
+      "rzls.linux-arm64": "vs-impl",
+      "rzls.linux-x64": "vs-impl",
+      "rzls.linux-musl-x64": "vs-impl",
+      "rzls.linux-musl-arm64": "vs-impl",
+      "rzls.neutral": "vs-impl",
+      "rzls.osx-arm64": "vs-impl",
+      "rzls.osx-x64": "vs-impl",
+      "rzls.win-arm64": "vs-impl",
+      "rzls.win-x64": "vs-impl",
+      "rzls.win-x86": "vs-impl",
+      "Microsoft.VisualStudio.DevKit.Razor": "vs-impl"
```

Comment:
Do we not need all of the platform specific versions of this?

Reply:
I don't believe so. Roslyn doesn't for theirs, and ours is less code. If we still want platform specific we can

Reply:
Roslyn also puts their telemetry DLL in their main package.. just saying 😛

Reply:
It ships a separate nuget package and downloads that into `.roslynDevKit`, which is what I'm planning to mimic for us